config/docker/clang-17-arm.jinja2

```diff
@@ -0,0 +1,13 @@
+{%- set sub_arch = 'armhf' %}
+{% extends 'clang-17.jinja2' %}
+
+{% block packages %}
+{{ super() }}
+
+RUN dpkg --add-architecture armhf
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    libgcc-10-dev:armhf
+
+{%- endblock %}
+
```

Comment:
This is exactly the same as `clang-16-arm.jinja2`, I think we can improve the templates to avoid duplication.

Reply:
Do you have any concrete suggestions for how to improve it, and do they need to block this merge? I'll note that I was just copying the existing idiom for this stuff.